pkg/cephmgr/osd/daemon.go

```diff
@@ -90,9 +104,56 @@ func getAvailableDevices(context *clusterd.DaemonContext, devices []*inventory.L
 			return nil, fmt.Errorf("failed to get device %s info. %+v", device.Name, err)
 		}
 		if fs == "" && ownPartitions {
-			available.Entries[device.Name] = &DeviceOsdIDEntry{Data: unassignedOSDID}
+			if desiredDevices == "all" {
```

Comment:
does this mean that only "all" will work to use devices (instead of a specific device name like "sda")